tasks.rb

```diff
@@ -514,6 +515,7 @@ def get_values(osw_file, step)
     step.setArgument('geometry_foundation_type', HPXML::FoundationTypeBasementConditioned)
     step.setArgument('geometry_foundation_height', 8.0)
     step.setArgument('geometry_foundation_height_above_grade', 1.0)
+    step.setArgument('geometry_rim_joist_height', 0.78)
```

Comment:
Something is not right if the rim joist height needs to be 0.78*12=9.36 inches.

Reply:
This is why I was asking about the dimensions of `base.xml`. At 30 x 45 (perimeter of 150), the rim joist would need to be 0.78 inches (0.78 x 150)  so that the rim joist area is 116 ft^2.

Reply:
Okay, this is resolved. The rim joist was 9.25 inches (2x10), but the area was rounded making me think it was 9.36 inches.